package.json

```diff
@@ -5,29 +5,30 @@
     "npm": "6.x.x"
   },
   "scripts": {
-    "lint": "eslint . --ignore-pattern public --ignore-pattern vendor",
-    "test": "NODE_ENV=test `npm bin`/mocha --require @babel/register --require spec/javascripts/spec_helper.js 'spec/javascripts/**/**_spec.js'"
+    "lint": "eslint . --ignore-pattern public --ignore-pattern vendor --ignore-pattern app/assets/builds",
+    "test": "NODE_ENV=test `npm bin`/mocha --require @babel/register --require spec/javascripts/spec_helper.js 'spec/javascripts/**/**_spec.js'",
+    "build": "webpack"
   },
   "dependencies": {
     "@rails/ujs": "^6.0.5",
-    "@rails/webpacker": "^4.3.0",
+    "fast-glob": "^3.2.7",
     "identity-style-guide": "^2.2.3",
-    "kind-of": "^6.0.3",
-    "minimist": "^1.2.6",
-    "serialize-javascript": "^3.1.0",
-    "webpack-merge": "^4.2.2"
+    "webpack": "^5.73.0",
```

Comment:
Do we need this to be a `dependencies` because we're only installing production dependencies in build environments?

- If yes, then we should also be sure to pull anything else required for build into `dependencies` (Babel, core-js)
- If no, then should we move this back to `devDependencies` ?

Reply:
FWIW, running `NODE_ENV=production rake assets:precompile` looks to result in an error because `jsbundling-rails` runs `yarn install` and prunes non-production dependencies.

```
$ webpack
assets by status 0 bytes [cached] 1 asset

ERROR in application
Module not found: Error: Can't resolve 'babel-loader' in '/Users/andrewmduthie/Documents/Code/identity-dashboard'
```

Dunno if that matches the expected behavior we'd have in deployed environments (can't confirm locally since `RAILS_ENV=production` is less than cooperatively locally).

Reply:
Yep, good catch. I've got a commit to move everything we need to build into dependencies.